src/block-components/typography/attributes.js

```diff
@@ -71,7 +71,7 @@ const typographyAttributes = {
 		default: false,
 	},
 	show: {
-		type: 'booleam',
```

Comment:
Because this is a change in the schema that might affect other blocks, can you check whether this affects blocks that use the `Typography` block component?

Can you try this:
1. Remove the change first (make it wrong again `booleam`)
2. Add blocks that use the `Typography` block component and change the text. if the block has a show toggle, add a block with the toggle off and another block with the toggle on.
3. Save the page
4. Add the fix in the schema (make it `boolean')
5. Then refresh the editor and see whether the blocks encounter an error.

If nothing errors, then this is good to go. If something errors, we'll need to handle that.